spec/system/task_spec.rb

```diff
@@ -2,23 +2,23 @@
 
 RSpec.describe 'Task', type: :system do
   describe 'Task一覧' do
+    let(:project) {create(:project)}
+    let(:task) {create(:task, project_id: project.id)}
```

Comment:
Factoryに`association`を設定することで`project_id`は省略可能ですね。
```suggestion
    let(:task) { create(:task) }
```